### PR TITLE
Fix breakage with Obsidian v15.4 and add customizable review buttons

### PR DESCRIPTION
**Pull request by AB1908**
The workspace methods have changed leading to a few funny bugs. This PR should address the important ones at least.

### CODE REVIEW

**Comment by AB1908** [comment]
Should fix #468.

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
         "husky": "^7.0.4",
         "jest": "^27.4.5",
         "moment": "^2.29.1",
-        "obsidian": "^0.13.11",
+        "obsidian": "^0.15.4",
```
Comment (fredriksy) [inline package.json:28]: The most recent update was obsidian 0.15.6 that caused the breaking change. Am I misunderstanding with this version being at 0.15.4?

Comment (AB1908) [reply package.json:28]: Perhaps. 15.4 was an insider release which changed `getActiveLeaf`. See release notes: https://discord.com/channels/686053708261228577/716028884885307432/993913278638084216

Comment (fredriksy) [reply package.json:28]: gotcha, that makes sense! thanks

**Comment by AB1908** [comment]
I've gone ahead and added a commit that closes #273 since it was an easy fix.

**Comment by st3v3nmw** [comment]
Hey @AB1908, thanks for the fixes! I'll add the localization support for the custom labels.

**Comment by AB1908** [comment]
Hey man! Happy to help out here, I love this plugin dearly and it's well written enough that I'm able to read and understand it. Thanks for all your work!

**Pull request merged**